docs/farming-&-staking/farming/space-acres/space-acres-install.mdx

```diff
@@ -0,0 +1,107 @@
+    
+    ![space-acres-install-1](/img/doc-imgs/space-acres/space-acres-install-1.png) 
+
+2. Choose your installation preferences, such as the destination folder and whether to create shortcuts. You can also select to have Space Acres launch at startup.
+
+    ![space-acres-install-2](/img/doc-imgs/space-acres/space-acres-install-2.png)
+
+3. Review your settings and click `Install` to commence the installation. The wizard will then install Space Acres on your system.
+
+    ![space-acres-install-3](/img/doc-imgs/space-acres/space-acres-install-3.png)
+
+4. The installation progress will be displayed, showing the status of new files being copied.
+
+    ![space-acres-install-4](/img/doc-imgs/space-acres/space-acres-install-4.png)
+
+5. Once the installation is complete, you will see a completion screen where you can choose to run Space Acres immediately by checking the box before clicking Finish.
+
+    ![space-acres-install-5](/img/doc-imgs/space-acres/space-acres-install-5.png)
+
+## Step 3: Initalize Space-Acres
+
+1. The welcome screen provides an overview of the necessary prerequisites for running Space Acres effectively. Make sure you meet all listed requirements before proceeding. For a comprehensive list of prerequisites and explanations, refer to our [prerequisites guide.](../prerequisites)  
+
+    ![space-acres-setup-1](/img/doc-imgs/space-acres/space-acres-setup-1.png)
+
+2. Configure your node and farmer settings by providing the following:
+    - Node path 
+    - Rewards address
+    - Farm path
+    - Farm size
+    - *(Optional) Add additional farms by selecting `Add farm`*
+
+    Click Start after the configuration is complete.
+
+    ![space-acres-setup-2](/img/doc-imgs/space-acres/space-acres-setup-2.png)
+
+3. (Optional) If you want to configure advanced settings, toggle the `Advanced configuration` section. This will allow you to set the following:
+    - Substrate Node P2P Port
+    - Subspace DSN P2P Port
+    - Enable Faster Networking (QUIC)
+
+    For more information on networking and its implications, refer to our [networking guide.](../additional-guides/networking)
+
+    ![space-acres-setup-3](/img/doc-imgs/space-acres/space-acres-setup-3.png)
+
+4. The application will then proceed to load, creating the consensus node and connecting to the network.
+
+    ![space-acres-setup-4](/img/doc-imgs/space-acres/space-acres-setup-4.png)
+
+5. Finally, you will see the main interface of Space Acres, where your consensus node is running and your farmer is progressing along. Initial syncing & plotting may take some time to complete. Below are examples of the various stages you can expect. 
+
+    a. Cache Syncing
+
+    ![space-acres-setup-5](/img/doc-imgs/space-acres/space-acres-setup-5.png) 
+
+    b. Initial Plotting
+
+    ![space-acres-setup-6](/img/doc-imgs/space-acres/space-acres-setup-6.png)
```

Comment:
I understand why those screenshots are different, but to me, it looks inconsistent to have a 10GB farm on one screenshot and a 2TB+ farm on the second one. It doesn't feel like a part of one process anymore.

Reply:
Regarding screenshots, I will have to do them all from scratch to resolve these issues, while I agree Its going to take some time to get a full "inline" set of screenshots as i will have to wait for complete node sync etc.

Reply:
Yeah, and with each drastic change in the UI, they will have to be changed as well. 

Let's create a kanban board task to keep that in mind next time we're spinning up another SA node.